port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -84,6 +84,20 @@ namespace port_drayage_plugin
         pt.put("cmv_id", _cmv_id);
         pt.put("cargo_id", _cargo_id);
         pt.put("operation", PORT_DRAYAGE_ARRIVAL_OPERATION_ID);
+
+        // Certain fields required by CARMA Streets are unnecessary for arrival messages and can be zero-filled
```

Comment:
Would it be cleaner to just omit certain fields or have different message types? Unless we're using some sort of JSON schema, it should be fine to only encode the fields that the receiving end is unpacking. If we need to strictly define these messages, it may be better to add presence bitstrings for the optional fields, to distinguish between empty data and actual zeros. Or perhaps consider using several different messages differentiated by a type field in the message.